en/docs/get-started/tutorials/create-your-first-event-triggered-integration.md

```diff
@@ -0,0 +1,122 @@
+
+1. Go to [https://console.choreo.dev/?profile=ipaas](https://console.choreo.dev/?profile=ipaas) and sign in to the Choreo Console.
+
+2. On the **Home** page, select a project or create a new one by clicking **+Create Project**.
+
+3. If the project has components, click **+Create**. If not, skip to the next step.
+
+4. On the **Event-Triggered Integration** card, click **Create** to start creating an event-triggered integration component
+
+5. Give the component a **name** and **description** (e.g. `MQTT listener` and `My first MI MQTT listener`).
+
+6. Click **Next**.
+
+7. Authorize Choreo to access your GitHub account by clicking **Authorize with GitHub**.
+
+8. If you have not already authorized Choreo applications, click **Authorize Choreo Apps** when prompted.
+
+9.  Enter information related to the GitHub repository you want to connect as follows:
+
+     | **Field**             | **Value**                                                                                             |
+     |-----------------------|-------------------------------------------------------------------------------------------------------|
+     | **GitHub Account**    | Select your GitHub account.                                                                           |
+     | **GitHub Repository** | Select your fork of the [choreo-examples GitHub repository](https://github.com/wso2/choreo-examples). |
+     | **Branch**            | `main`                                                                                                |
+
+10. Under **Build Preset**, click **Micro Integrator**.
+
+     !!! info
+        Build Preset specifies the type of build that must be executed, based on the implementation of the component. For instance, if the component was developed using [WSO2 Integration Studio](https://wso2.com/integration/integration-studio/), the Micro Integrator build preset should be chosen. Meanwhile, if the Ballerina language was used to develop the component, the Ballerina build preset should be selected. These presets convert the integration code into a Docker image that can run on Choreo Cloud.
+
+
+11. Enter information as follows.
+
+     | **Field**             | **Value**                                      |
+     |-----------------------|------------------------------------------------|
+     | **Path**              | `ipaas/wso2-synapse/simple-rabbitmq-listener`  |
+
+
+    !!! info
+        When Choreo builds the component this is the path it looking for the project in the repository.
+ 
+1.  Click **Create**.
+
+
+## Step 2: Deploy
+Before deploying the component you need to assign RabbitMQ credentials to environment variables by following the steps given below:
+
+1. Go to **Deploy** In the left navigation menu.
+2. Open the `DevOps Portal` by Clicking **Configure via the DevOps Portal** in **Development** card.
+3. In DevOps Portal, Add the following environment variables and save.  
```

Comment:
Since this is going to be a getting started type tutorial for anyone who wants to explore creating and managing event-triggered integrations in Choreo, would be good to specify the exact steps with regard to getting the scenario in the tutorial completed. Pointing off to the DevOps doc for more details is ok, but here we need to mention the steps like which configuration type to go with  and so on. WDYT?

Reply:
Agreed. it would complete that step of the tutorial. perhaps we can remove or update if DevOps Doc/UI-UX is changed later.